make/Images.gmk

```diff
@@ -108,6 +109,55 @@ $(eval $(call SetupExecute, jlink_jdk, \
 
 JLINK_JDK_TARGETS := $(jlink_jdk)
 
+ifeq ($(JLINK_PRODUCE_RUNTIME_LINK_JDK), true)
+  RL_BUILD_CLASSES := runtimeimagelinkdeltaproducer-classes
+  RL_DELTA_GEN_CLASSES := $(BUILDTOOLS_OUTPUTDIR)/$(RL_BUILD_CLASSES)
+
+  RL_JIMAGE_PATH_ARG := $(JDK_IMAGE_DIR)/lib/modules
+  RL_DIFFS_OUTPUT_FILE_ARG := $(JDK_IMAGE_DIR)/lib/runtime-image-link.delta
+  RL_MOD_PATH_ARG := $(IMAGES_OUTPUTDIR)/jmods
+  TOOL_RUNTIME_IMAGE_LINK_DELTA_PRODUCER := $(BUILD_JAVA) --add-modules jdk.jlink \
```

Comment:
All of these are only used once so should rather be inlined. I think that makes it easier to understand and read.